### DISCUSSION TITLE
Welcome to react-native-paper Discussions!

### DISCUSSION THREAD

**Discussion by Trancever** [callstack/react-native-paper#2403] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community. You can also share with us what you're building with Paper!

**Comment by nzmedet** [comment]
Do you guys have plan on making dropdown or select or picker components?

**Reply by b2d3** [reply]
Agree! Whole library is awesome by the way, but an input selector would be great.

**Reply by alexej-strelzow** [reply]
I'd love to have sth like this: https://www.npmjs.com/package/react-native-autocomplete-input

**Comment by vesolovski** [comment]
I love your library! Great job! Thanks! The problem I have is that I can use it on codesandbox.io in version 3.12.0, but any higher version crashed (vide https://codesandbox.io/s/please-fix-react-native-paper-or-codesandbox-5wqw5q - please try higher version of react-native-paper to see what I mean). Any hints on that issue, please?

ERROR DISPLAYED:
TypeError: Cannot read properties of undefined (reading 'Left') at $csb$eval (https://5wqw5q.csb.app/node_modules/react-native-paper/lib/module/components/TextInput/Adornment/Affix.js:27:41)

Another problem is that icons are not being displayed by codesandbox.io (which I prefer over expo.dev) but this time the problem is -  I believe - with codesandbox.io site. Any help would be much appreciated :)

**Comment by batazo** [comment]
My solution is I chacged the button component to the simple React-Native button and I styled it
